docs/src/main/paradox/client/walkthrough.md

```diff
@@ -26,19 +26,39 @@ enablePlugins(AkkaGrpcPlugin)
 Gradle
 :  @@@vars
 ```gradle
+// settings.gradle
+pluginManagement {
+  repositories {
+    gradlePluginPortal()
+    maven {
+      url "https://repo.akka.io/maven"
+    }
+  }
+  plugins {
+    id 'com.lightbend.akka.grpc.gradle' version '$project.version$'
+  }
+}
```

Comment:
I _think_ I tried that and it didn't work, but I may be wrong. Can check tomorrow.

Reply:
Dropping buldscript seems to work: https://github.com/akka/akka-grpc-quickstart-java.g8/pull/64

Reply:
Single file seems nicer though, I'll try only buildscript and no settings.gradle as well

Reply:
Good, once we got it working it might be worth changing plugin-tester projects so that they look the same. There I think we need the local repo as well, since we are testing with a local publish.

Reply:
Updated the builds here, slightly simpler in some sense, but had to keep the gradle.settings to include local maven in  repositories searched for plugins.